site/en/blog/new-in-chrome-94/index.md

```diff
@@ -0,0 +1,183 @@
+    A Babylon.js demo of a rough sea being simulated using WebGPU's compute
+    shader capability.
+  </figcaption>
+</figure>
+
+Francois has a great article [Access modern GPU features with WebGPU][wd-gpu]
+on web.dev with the details, and compares the performance of matrix
+multiplication running on the CPU vs the GPU. Here's a hint, the
+[GPU wins][wd-compute-perf].
+
+## PWA Summit {: #pwa-summit }
+
+The [PWA Summit][pwa-summit] is coming up October 6-7. It's a
+a free, online conference focused on helping everyone succeed with Progressive
+Web Apps. The PWA Summit is a collaboration between folks from a handful of
+different companies involved in the creation of PWA technologies: Google,
+Intel, Microsoft, and Samsung.
+
+There are a ton of great talks and content. You can learn more and register at
+[PWASummit.org][pwa-summit]. I hope to see you there!
+
+## And more! {: #more }
+
+Of course there's plenty more.
+
+The prioritized [`scheduler.postTask()`][cs-scheduler] method allows you to
+schedule tasks, and dynamically change their priorities, or cancel them all
+together.
+
+If you've ever fought with a re-layout when scroll bars appear, the
+[`scrollbar-gutter`][cs-gutters] property will make you happy. It provides
+control over the presence of scrollbar gutters, allowing you to prevent
+layout changes as content expands.
+
+The use of [WebSQL in **third-party** contexts is now deprecated][websql-3p],
+and removal is expected in Chrome 97. Web SQL Database standard was abandoned
+in November 2010. It was never implemented in Firefox, and was deprecated in
+Safari in 2019. It will be deprecated and removed from Chrome when usage is
+low enough. If you're still using WebSQL, now would be a good time to start
+planning your migration off of it.
+
+And the [virtual keyboard API][wd-vkeyboard] gives you more control over how
+and when the virtual, on-screen keyboard is shown. It allows you to explicitly
+control any scrolling behaviour, or changes to layout when the keyboard
+appears or disappears.
+
+## Further reading
+
+This covers only some of the key highlights. Check the links below for
+additional changes in Chrome 94.
+
+* [What's new in Chrome DevTools (94)](/blog/new-in-devtools-94/)
+* [Chrome 94 deprecations & removals](/blog/deps-rems-94/)
+* [ChromeStatus.com updates for Chrome 94](https://www.chromestatus.com/features#milestone%3D94)
+* [What's new in JavaScript in Chrome 94](https://v8.dev/blog/v8-release-94)
+* [Chromium source repository change list](https://chromium.googlesource.com/chromium/src/+log/93.0.4577.69..94.0.4606.56)
+* [Chrome release calendar](https://chromiumdash.appspot.com/schedule)
+
+## Subscribe
+
+To stay up to date, [subscribe](https://goo.gl/6FP1a5)
+to [Chrome Developers YouTube channel](https://www.youtube.com/user/ChromeDevelopers/),
+and you'll get an email notification whenever we launch a new video.
+
+I'm Pete LePage, and as soon as Chrome 95 is released, I'll be right here to
```

Comment:
did you want to add a personal blog link here as well? :)

Reply:
Nah - thats fine.